src/openrct2/object/ObjectRepository.cpp

```diff
@@ -128,13 +129,21 @@ class ObjectFileIndex final : public FileIndex<ObjectRepositoryItem>
         stream->WriteValue(item.ObjectEntry);
         stream->WriteString(item.Path);
         stream->WriteString(item.Name);
+
         uint8_t sourceLength = static_cast<uint8_t>(item.Sources.size());
         stream->WriteValue(sourceLength);
         for (auto source : item.Sources)
         {
             stream->WriteValue(source);
         }
 
+        uint8_t authorsLength = static_cast<uint8_t>(item.Authors.size());
+        stream->WriteValue(authorsLength);
+        for (auto author : item.Authors)
```

Comment:
```suggestion
        for (const auto& author : item.Authors)
```
To prevent unnecessary copy